src/workerd/server/workerd.capnp

```diff
@@ -386,6 +386,17 @@ struct Worker {
       unsafeEval @23 :Void;
       # A simple binding that enables access to the UnsafeEval API.
 
+      volatileCache :group {
+        # A binding representing access to an in-memory cache.
+
+        id @24 :Text;
```

Comment:
Can this be optional, if you don't set it you get a cache that isn't shared with anyone?

Reply:
Probably. At least initially I wanted to make this a straightforward port of the internal impl without any changes. We could mark this experimental for now, make any changes we want to improve, then unflag?

Reply:
Made optional now.